projects/map/src/lib/openlayers-map/openlayers-map.ts

```diff
@@ -139,15 +139,8 @@ export class OpenLayersMap implements MapViewerModel {
     const geom = olFeature.getGeometry();
     if (geom) {
       const geomExtent = geom.getExtent();
-      this.getVisibleExtent$()
-        .pipe(take(1))
-        .subscribe(mapExtent => {
-          if (containsExtent(mapExtent, geomExtent)) {
-            return;
-          }
-          const center = getCenter(geomExtent);
-          this.zoomTo(center[0], center[1]);
-        });
+      const center = getCenter(geomExtent);
+      this.zoomTo(center[0], center[1]);
```

Comment:
this doesn't zoom the map to the feature, it only moves the feature into the middle of the view; the zoomLevel stays the same, so small features are practically invisible when selected in the list.

I think the existing `OpenLayersMap#zoomToFeature(olFeature: Feature<Geometry>)` should be fixed to call [View#fit(...)](https://openlayers.org/en/latest/apidoc/module-ol_View-View.html#fit)

Unfortunately the story is unclear how much the view should zoom in... but you could:
- get the extent of the feature
- buffer that with n*times the largest axis of the extent
- use that to fit the view

for point features and very small features the buffer would probably need to be larger of you would need a minimum threshold for the extent

Reply:
OK; 10 will work for any map projection that uses meter as the UOM, but not for eg. WGS84/degrees, so we will have to revisit